### PR TITLE
UI edits re: VCF updates -- SearchResultTableColumns

### PR DESCRIPTION
**Pull request by alexkb0009**
Self-explanatory -- changes SearchResultTable VariantSample to look bit more like was before.

### CODE REVIEW

File: src/encoded/static/components/item-pages/VariantSampleView/VariantSampleInfoHeader.js [hunk 1]
```diff
@@ -107,7 +107,7 @@ function TranscriptSelectionSection(props){
 
     const geneListOptions = geneTranscriptList.map(function(transcript, idx){
         return (
-            <DropdownItem key={idx} eventKey={idx} active={idx === currentTranscriptIdx}>
+            <DropdownItem key={idx} eventKey={idx.toString()} active={idx === currentTranscriptIdx}>
```
Comment (alexkb0009) [inline src/encoded/static/components/item-pages/VariantSampleView/VariantSampleInfoHeader.js:110]: Might be redundant re: Bianca's updates/branch; hopefully merge will converge em automatically.

Comment (Bianca-Morris) [reply src/encoded/static/components/item-pages/VariantSampleView/VariantSampleInfoHeader.js:110]: No worries, I figured you had added this to your branch, so didn't commit anything to mine on this note.

**Review by Bianca-Morris — APPROVED** [review]
Looks good. I do wonder about the minor style changes to bamsnap link (didn't load this up on local), but I figure if anyone takes issue with it we can address it then.

File: src/encoded/ingestion/table_utils.py [hunk 2]
```diff
@@ -496,7 +496,7 @@ def extend_variant_sample_columns(cols):
         if "variant.genes.genes_most_severe_gene.display_title" in cols:
             # We combine `genes_most_severe_gene` + `genes_most_severe_transcript` columns in the UI column render func for compactness.
             cols["variant.genes.genes_most_severe_gene.display_title"].update({
-                "colTitle": "Gene, Transcript",
+                "title": "Gene, Transcript",
```
Comment (alexkb0009) [inline src/encoded/ingestion/table_utils.py:499]: Changed from colTitle to title because "title" is what's visible when hover over header title.

**Review by willronchetti — APPROVED** [review]

**Review by utku-ozturk — APPROVED** [review]

**Pull request merged**